documentation/projects/proposals/trust_and_safety/content_report_moderation/20230517-implementation_plan_mature_copy_change.md

```diff
@@ -0,0 +1,235 @@
+- `test/playwright/e2e/report-media.spec.ts`
+- `test/playwright/e2e/search-query-server.spec.ts`
+- `test/unit/fixtures/audio.js`
+- `test/unit/specs/components/v-content-report-form.spec.js`
+- `test/unit/specs/stores/search-store.spec.js`
+- `test/unit/specs/utils/search-query-transform.spec.js`
+
+The following objects within the codebase will need to be renamed (along with
+any references that depend on them):
+
+- [`Filters` interface](https://github.com/WordPress/openverse/blob/f676e57e2dbe9e92f568d413ade83ce6c2f38f9b/frontend/src/constants/filters.ts#L28)
+- [`NonMatureFilterCategory` type](https://github.com/WordPress/openverse/blob/f676e57e2dbe9e92f568d413ade83ce6c2f38f9b/frontend/src/constants/filters.ts#L31)
+- [`mediaFilterKeys` constant references](https://github.com/WordPress/openverse/blob/f676e57e2dbe9e92f568d413ade83ce6c2f38f9b/frontend/src/constants/filters.ts#L37)
+- [`filterCodesPerCategory` constant](https://github.com/WordPress/openverse/blob/f676e57e2dbe9e92f568d413ade83ce6c2f38f9b/frontend/src/constants/filters.ts#L86)
+- [Content report constant](https://github.com/WordPress/openverse/blob/6baeb6b41430a586d4ed6c6976b7bae8ba1167fa/frontend/src/constants/content-report.ts#L2)
+- References to `Fake ~50% of results as mature` in
+  [`src/stores/media/single-result.ts`](https://github.com/WordPress/openverse/blob/74da3d852a45c514c0159038c86164a26d70504b/frontend/src/stores/media/single-result.ts#L122)
+  and
+  [`src/stores/media/index.ts`](https://github.com/WordPress/openverse/blob/74da3d852a45c514c0159038c86164a26d70504b/frontend/src/stores/media/index.ts#L432).
+- [Search store references](https://github.com/WordPress/openverse/blob/92ea64c6d4ac1801785eadf99a3782628eadad06/frontend/src/stores/search.ts)
+- [Content report utilities](https://github.com/WordPress/openverse/blob/74da3d852a45c514c0159038c86164a26d70504b/frontend/src/utils/content-safety.ts#L10-L23)
+- All affected tests.
+
+The following references _will not be_ affected because they correspond to
+either the data model or to the API query parameter `mature`. The query
+parameter will be changed to `include_sensitive_results` in the API
+[as part of another project](https://docs.openverse.org/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230309-project_proposal_detecting_sensitive_textual_content.html#filtering)
+
+- [Search query transform utilities](https://github.com/WordPress/openverse/blob/92ea64c6d4ac1801785eadf99a3782628eadad06/frontend/src/utils/search-query-transform.ts#L31)
+  _(query parameter)_
+- [`Media` interface attributes](https://github.com/WordPress/openverse/blob/930a2ec4583d53e2f17e7aa397dc9b3b32cc8ebb/frontend/src/types/media.ts#L48)
+  _(data model)_
+
+#### Copy
+
+Almost all copy changes necessary can be made in the
+[`en.json5` base translations file](https://github.com/WordPress/openverse/blob/2041c5df1e9d5d1f9f37e7c177f2e70f61ea5dba/frontend/src/locales/scripts/en.json5#L1-L0).
```

Comment:
```suggestion
[`en.json5` base translations file](https://github.com/WordPress/openverse/blob/2041c5df1e9d5d1f9f37e7c177f2e70f61ea5dba/frontend/src/locales/scripts/en.json5#L487-L491).
```

Reply:
Ah, I meant to link the whole file here, will fix!